runtime/common/address.go

```diff
@@ -31,6 +31,9 @@ const AddressLength = 8
 
 type Address [AddressLength]byte
 
+// Singleton for address 0x0. This should be used in place of an empty initializer `Address{}`
+var NilAddress Address = Address{}
```

Comment:
Nitpick: this is more of a "ZeroAddress" than a "NilAddress", can we use that name instead?

Reply:
As 0x0 is invalid address, I think NilAddress makes more sense here. 

Though I believe `Address{}` was also clear, I think we can maybe also alias `Stack` here. Most of the code usage of `Address{}` is creating something on stack I believe.